spec/features/agent_manages_appointments_spec.rb

```diff
@@ -190,6 +221,59 @@ def when_they_want_to_book_an_appointment
     @page = Pages::NewAppointment.new.tap(&:load)
   end
 
+  def when_they_accept_the_appointment_preview
+    @page = Pages::PreviewAppointment.new
+    expect(@page).to be_displayed
+    @page.confirm_appointment.click
+  end
+
+  def when_they_go_back_to_edit_their_appointment
+    @page = Pages::PreviewAppointment.new
+    expect(@page).to be_displayed
+    @page.edit_appointment.click
+  end
+
+  def and_they_change_some_details
+    @page = Pages::NewAppointment.new
+
+    @page.first_name.set                 'Another'
```

Comment:
Not a huge fan of this alignment, to me at least it's actually more difficult to scan / read. Do you find it much easier? (not a blocker, just a question)